app/views/users/show.html.erb

```diff
@@ -7,5 +7,5 @@
       <p><%= post.text %><p>
     </div>
   <% end %>
- 
+ <a href="<%= user_posts %>" class="btn btn-primary justify-content-center mt-3"> See all post</a>
 </div>
```

Comment:
- [ ] Great job on creating the views, however, clicking on each user should take the user to the user show page. This is not the case, an error occurs, kindly fix this.